app/Controllers/authController.php

```diff
@@ -141,12 +145,12 @@ public function formLoginAction() {
 
 				// All is good, go back to the index.
 				Minz_Request::good(_t('feedback.auth.login.success'),
-				                   array('c' => 'index', 'a' => 'index'));
+								   array('c' => 'index', 'a' => 'index'));
```

Comment:
There is still a strange mix of tabs and spaces, isn't it?

Reply:
done

Reply:
This line was only an example :-)
Please have a look from the GitHub UI https://github.com/FreshRSS/FreshRSS/pull/1751/files

Reply:
how do you spot those strange mix. I don't know if we are talking about the same thing. I've looked in the UI you linked and I am not sure what to look for.

Reply:
Here is a screenshot, just to be sure we look at the same thing :-)

![image](https://user-images.githubusercontent.com/1008324/34691997-d1b3ff10-f4be-11e7-94f6-cb47e43342cf.png)

I see a lot of replaced whitespace, which have kept the same apparent indenting, but produced a git change anyway, which means that there has been a tab <-> space conversion.

Just putting the mouse on one of those lines shows that it is a mixture of tabs and space.

My local text editor also shows a difference (showing white spaces is quite common in various text editors).

It is even more visible when the tab size is different than 4, such as the default value of 8 used in gedit:

![image](https://user-images.githubusercontent.com/1008324/34692503-b5006884-f4c0-11e7-9c6a-7d02b1d3e374.png)

To be a bit more systematic, a regex search with something like `\t[ ][^*]` should highlight them.

Reply:
Ok, I see what you mean. Thank you

Reply:
@Alkarex So I've tried to restore the old spacing, but it doesn't work with phpcs. Here is the error I've got if I try to use spaces instead of tabs.

    -------------------------------------------------------------------------------------------------------
    FOUND 3 ERRORS AFFECTING 3 LINES
    -------------------------------------------------------------------------------------------------------
     153 | ERROR | [x] Tabs must be used to indent lines; spaces are not allowed
     154 | ERROR | [x] Tabs must be used to indent lines; spaces are not allowed
     155 | ERROR | [x] Tabs must be used to indent lines; spaces are not allowed
    -------------------------------------------------------------------------------------------------------
    PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
    -------------------------------------------------------------------------------------------------------

I don't know what to do now. On one hand I want to keep phpcs running smoothly, on the other end, it is not configured to allow what we defined for the coding standards.
If you have any idea, that would be appreciated.